pkg/endpointmanager/manager.go

```diff
@@ -343,3 +343,18 @@ func AddEndpoint(owner endpoint.Owner, ep *endpoint.Endpoint, reason string) err
 
 	return nil
 }
+
+// WaitForEndpointsAtPolicyRev waits for all endpoints to be at a given policy
+// revision.
```

Comment:
Please clarify the comment to say:

... waits for all endpoints which existed at the time this function is called to be at a given policy revision. New endpoints appearing while waiting are ignored.